samples/Meeting/src/app/views/EndCall.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can we also remove this page - the meeting composite has an end call page.
Reason being we want to make sure we are testing the end call page of the composite when running locally